docs/docker-image-build.md

```diff
@@ -0,0 +1,29 @@
+# Build a `ks` docker image
+
+The intended use case for building a docker images is to run `ks` from CI/CD. As a development tool it is recommended to install `ks` locally, though it is still possible to use the docker image for development as well.
+
+Copy and paste the following commands:
+```bash
+# Clone the ksonnet repo into your GOPATH
+go get github.com/ksonnet/ksonnet
+
+# Build and install binary under shortname `ks` into $GOPATH/bin
+cd "${GOPATH}/src/github.com/ksonnet/ksonnet"
+make docker-image
+```
+
+# Running `ks` via docker
+
+In order to run via docker, the `ks` process needs access to a kubernetes config file, certificates, and the current working directory. We can set these up using mounts. Here's what it looks like running on a local cluster with minikube:
```

Comment:
Instead of "We can set these up using mounts"

Pass the `--mount` flag to capture the path of these files.